core/src/main/kotlin/org/lflang/generator/cpp/CppAssembleMethodGenerator.kt

```diff
@@ -100,6 +100,12 @@ class CppAssembleMethodGenerator(private val reactor: Reactor) {
         }
     }
 
+    private val Connection.cppDelay: String 
+        get() = if (delay != null) "${delay.toCppTime()}" else "0s"
```

Comment:
You can use the elvis operator for this. Also the magic string is redundant.
```suggestion
        get() = delay?.toCppTime() ?: "0s"
```